src/modm/platform/spi/bitbang/bitbang_spi_master_impl.hpp

```diff
@@ -4,6 +4,7 @@
  * Copyright (c) 2012-2017, Niklas Hauser
  * Copyright (c) 2014, Sascha Schade
  * Copyright (c) 2018, Raphael Lehmann
+ * Copyright (c) 2020, Erik Henriksson
```

Comment:
And what did you change? This giant diff really isn't selling me on using clang format and basically just demotivates me to maintain modm entirely. Like, how do you expect me to review this?

Reply:
Much, much better now. Thank you!